src/test/java/com/codurance/retropolis/controllers/CardControllerTest.java

```diff
@@ -202,7 +195,7 @@ public void return_bad_request_when_addVote_is_empty_on_add_vote() throws Except
     assertEquals("addVote cannot be empty", errorResponse.get(0));
   }
 
-  //TODO Refactor
+  //TODO #1. Refactor: maybe extract this function to a separate file
```

Comment:
remove the todos and create an issue

Reply:
I 'll create an issue, but do you mind keeping the TODO on the codebase and remove it when the issue is actually resolved?